cdm/src/main/java/ucar/nc2/ft2/coverage/simpgeometry/CFPolygon.java

```diff
@@ -96,11 +96,23 @@ public void setData(Array data) {
 		this.data = data;
 	}
 	
+	/**
+	 * Sets the next polygon which make up the multipolygon which this polygon is a part of.
+	 * If next is a CFPolygon, automatically connects the other polygon to this polygon as well.
+	 */
+	public void setNext(Polygon next) {
+		if(next instanceof CFPolygon) {

```

Comment:
good use of `instanceof`

Reply:
Thanks! One of those rare uses of it, cast safety.